tests/resources/test_gemd_resource.py

```diff
@@ -469,3 +472,96 @@ def test_batch_delete(gemd_collection, session):
 def test_batch_delete_bad_input(gemd_collection):
     with pytest.raises(TypeError):
         gemd_collection.batch_delete([False])
+
+
+def test_type_passthrough(gemd_collection, session):
```

Comment:
I think these two tests need some comments explaining what they're doing. It is far from clear to the naive reader.

Reply:
```suggestion
def test_type_passthrough(gemd_collection, session):
    """Verify objects that are not directly referenced by objects (e.g., a tuple of Templates) don't get type information stripped."""
```